PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install Remote Assist.md

```diff
@@ -11,52 +11,53 @@ mac
 ```
 set -euo pipefail
 
-declare -r REMOTE_DMG_URL="https://cdn.jumpcloud.com/TheJumpCloud/jumpcloud-remote-assist-agent/latest/jumpcloud-assist-app.dmg"
-declare -r LOCAL_DMG_TMP_PATH="/tmp/jumpcloud-assist-app.dmg"
-declare -r APP_NAME="Jumpcloud Assist App"
-declare -r APP_FILENAME="$APP_NAME.app"
+declare -r REMOTE_PKG_URL="https://cdn.jumpcloud.com/TheJumpCloud/jumpcloud-remote-assist-agent/latest/jumpcloud-remote-assist.pkg"
+declare -r LOCAL_PKG_TMP_PATH="$(mktemp -d)/jumpcloud-remote-assist.pkg"
 
 function get_app_pid() {
-    ps -ef | awk "/$APP_NAME$/ {print \$2}"
+    local -r APP_NAME=$1
+    ps -eo pid,comm | awk "/$APP_NAME$/ {print \$1}"
 }
 
-if PID=$(get_app_pid) && [[ -n "$PID" ]]; then
-    kill -s TERM "$PID"
-    sleep 1
-fi
+function kill_app_by_name() {
+    local -r APP_NAME=$1
+    if PID=$(get_app_pid "$APP_NAME") && [[ -n "$PID" ]]; then
+        kill -s TERM "$PID"
+        sleep 1
+    fi
 
-if PID=$(get_app_pid) && [[ -n "$PID" ]]; then
-    kill -s KILL "$PID"
-fi
+    if PID=$(get_app_pid "$APP_NAME") && [[ -n "$PID" ]]; then
+        kill -s KILL "$PID"
+    fi
+}
+
+# Clean up installs having legacy names
+kill_app_by_name "Jumpcloud Assist App"
```

Comment:
Do we also need to remove the legacy name application files?

Reply:
No, it is not in the AC for TX-396.